### PR TITLE
Rebase 4.14.0 0.nightly 2023 03 08 194110 amd64 2023 03 08 arm64 2023 03 09

### PR DESCRIPTION
**Pull request by pmtk**

### CODE REVIEW

File: scripts/auto-rebase/rebase.sh [hunk 1]
```diff
@@ -908,8 +903,8 @@ rebase_to() {
     local lvms_operator_bundle_manifest=$3
 
     title "# Rebasing to ${release_image_amd64} and ${release_image_arm64}"
-    download_release "${release_image_amd64}" "${release_image_arm64}"
-    download_lvms_operator_bundle_manifest "$lvms_operator_bundle_manifest"
+    #download_release "${release_image_amd64}" "${release_image_arm64}"
```
Comment (dhellmann) [inline scripts/auto-rebase/rebase.sh:906]: I think some of these changes were probably for debugging?

File: scripts/auto-rebase/assets.yaml [hunk 2]
```diff
@@ -45,12 +45,7 @@ assets:
       - file: topolvm-csi-provisioner_rbac.authorization.k8s.io_v1_clusterrolebinding.yaml
       - file: topolvm-csi-provisioner_rbac.authorization.k8s.io_v1_role.yaml
       - file: topolvm-csi-provisioner_rbac.authorization.k8s.io_v1_rolebinding.yaml
-      - file: topolvm-controller_rbac.authorization.k8s.io_v1_role.yaml
```
Comment (dhellmann) [inline scripts/auto-rebase/assets.yaml:48]: In #1481 I had to update the lvms component to not try to load these assets. You're also removing more files than I removed. Were some of these already unused?

**Review by dhellmann — COMMENTED** [review]
cc @copejon

File: scripts/auto-rebase/rebase.sh (hunk 1 above)
Comment (pmtk) [reply scripts/auto-rebase/rebase.sh:906]: haha, yes :laughing:

File: scripts/auto-rebase/assets.yaml (hunk 2 above)
Comment (pmtk) [reply scripts/auto-rebase/assets.yaml:48]: I looked into this again, some of lvms manifests in our repo had names changed and it broke the rebase

Comment (pmtk) [reply scripts/auto-rebase/assets.yaml:48]: Actually, I might just bring them back and let rebase job do its job so we get nice changelog in PR's description

**Pull request closed without merging**